src/vstest.console/CommandLine/Executor.cs

```diff
@@ -82,6 +83,9 @@ public Executor(IOutput output) : this(output, TestPlatformEventSource.Instance,
 
     internal Executor(IOutput output, ITestPlatformEventSource testPlatformEventSource, IProcessHelper processHelper, IEnvironment environment)
     {
+        DebuggerBreakpoint.AttachVisualStudioDebugger("VSTEST_RUNNER_DEBUG_ATTACHVS");
```

Comment:
That would be ideal, but unfortunately there's no good place to define those constants if they are to be used throughout the codebase. Also, in this particular case this is the only relevant place the constant is being used (it is also used in tests but that doesn't really count since tests will start to break should we ever change this name).